assets/js/components/dashboard/dashboard-app.js

```diff
@@ -39,6 +43,21 @@ class DashboardApp extends Component {
 				<div className="googlesitekit-module-page">
 					<div className="mdc-layout-grid">
 						<div className="mdc-layout-grid__inner">
+							<div className="
+								mdc-layout-grid__cell
+								mdc-layout-grid__cell--span-10-desktop mdc-layout-grid__cell--span-6-tablet mdc-layout-grid__cell--span-2-phone
+							">
+								<PageHeader
+									className="googlesitekit-page-header__page-title"
+									title={ __( 'Site Overview', 'google-site-kit' ) }
+								/>
+							</div>
+							<div className="
+									mdc-layout-grid__cell
+									mdc-layout-grid__cell--span-2-desktop mdc-layout-grid__cell--span-2-tablet mdc-layout-grid__cell--span-2-phone mdc-layout-grid__cell--align-top mdc-layout-grid__cell--align-right
```

Comment:
Can you replace the `mdc-layout-grid__cell--align-top` class here with `mdc-layout-grid__cell--align-middle` to make the dropdown vertically center-aligned?